sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -70,6 +70,11 @@ public virtual string AccountName
             }
         }
 
+        /// <summary>
+        /// The Uri of the Table.
```

Comment:
Is Table intended to be capitalized?   Seems the rest of the comments use it with lower case.